src/state/GlobalState.tsx

```diff
@@ -266,21 +266,9 @@ export class GlobalState extends PureComponent<Props> {
   }
 
   checkVersion = async () => {
-    const { t } = this.props
     const newVersion = await ipcRenderer.invoke('checkVersion')
     if (newVersion) {
-      const { response } = await ipcRenderer.invoke('openMessageBox', {
```

Comment:
In this case, what happens on the other versions besides appImage?
I also forgot to ask if the information about the updates on the installable version. How are we handling the check if the user is using appImage or has installed heroic in the system?

Reply:
See https://stackoverflow.com/questions/66974771/see-build-target-of-electron-builder-in-running-application

Closing